docs/source/pages/using.rst

```diff
@@ -314,7 +314,53 @@ Analyzing a locally cloned repository
 
 If you have a local repository that you want to analyze, Macaron also supports running the analysis against a local repository.
 
-Assume that the dir tree at the local repository has the following components:
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+Analyzing a repository whose git service is not supported by Macaron
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+If the repository remote URL is from an unknown git service (see :ref:`Git Services <supported_git_services>` for a list of supported git services in Macaron), Macaron won't recognize it when analyzing the repository.
+
+You would need to tell Macaron about that git service through the ``defaults.ini`` config.
+For example, let's say you want to analyze a repository hosted at ``https://git.example.com/foo/target``. First, you need to create a ``defaults.ini`` file in the current workspace with the following content:
+
+.. code-block:: ini
+
+  [git_service.local_repo]
+  hostname = git.example.com
+
+In which ``hostname`` contains the domain of the git service URL. In this example it's ``git.example.com``.
```

Comment:
```suggestion
In which ``hostname`` contains the hostname of the git service URL. In this example, it is ``git.example.com``.
```

The term "domain" used here may not be the most accurate. We had something similar to this before. See #416.

Reply:
Fixed in https://github.com/oracle/macaron/pull/694/commits/ace778c0b7ee12da54461140ddee63ff2ebb1f13 and https://github.com/oracle/macaron/pull/694/commits/a96ab2bbd8d6ff29f32e30b77619e8326cc31fa9